src/StructuredLogger/BinaryLogger/BinaryLogReplayEventSource.cs

```diff
@@ -57,7 +57,7 @@ internal interface IBinaryLogReplaySource :
     /// by implementing IEventSource and raising corresponding events.
     /// </summary>
     /// <remarks>The class is public so that we can call it from MSBuild.exe when replaying a log file.</remarks>
-    internal sealed class BinaryLogReplayEventSource : EventArgsDispatcher, IBinaryLogReplaySource
+    public sealed class BinaryLogReplayEventSource : EventArgsDispatcher, IBinaryLogReplaySource

```

Comment:
I don't think we can make this public because it will then conflict with the one in MSBuild (so consumers referencing both us and MSBuild will have a conflict of both).